tests/ColorDistanceLineTraceTest.cpp

```diff
@@ -0,0 +1,286 @@
+/**
+ * @file   ColorDistanceLineTraceTest.cpp
+ * @brief  ColorDistanceLineTraceクラスのテスト
+ * @author miyahara046
+ */
+
+#include "ColorDistanceLineTrace.h"
+#include <gtest/gtest.h>
+#include <gtest/internal/gtest-port.h>
+
+using namespace std;
+
+namespace etrobocon2025_test {
+  // 最初3回の色取得で連続して指定色を取得するテストケース
+  TEST(ColorDistanceLineTraceTest, runToGetFirst)
+  {
+    Robot robot;
+    COLOR targetColor = COLOR::GREEN;
+    double targetDistance = 10000.0;
+    double targetSpeed = 50000.0;
```

Comment:
結構非現実的な値じゃない？
```suggestion
    double targetDistance = 1000.0;
    double targetSpeed = 500.0;
```
とかが妥当かなー。他のところも。